tests/phpunit/agentic-commerce/class-wc-stripe-agentic-commerce-manual-approval-test.php

```
@@ -0,0 +1,353 @@
+			function ( $reason, $evt ) use ( &$captured_event ) {
+				$captured_event = $evt;
+				return $reason;
+			},
+			10,
+			2
+		);
+
+		$this->approval->validate( $event );
+
+		$this->assertInstanceOf( WC_Stripe_Agentic_Customize_Checkout_Event::class, $captured_event );
+	}
```

PHPMD (2.15.0)

[error] The method test_filter_receives_event is not named in camelCase. (undefined)

(CamelCaseMethodName)